File: 백준/Bronze/9063. 대지/대지.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        int minX = 10000;
        int maxX = -10000;
        int minY = 10000;
        int maxY = -10000;

        int n = sc.nextInt();

        for (int i = 0; i < n; i++) {
            int x = sc.nextInt();
            int y = sc.nextInt();

            minX = Math.min(minX, x);
            maxX = Math.max(maxX, x);
            minY = Math.min(minY, y);
            maxY = Math.max(maxY, y);
        }

        int area = (maxX - minX) * (maxY - minY);
        System.out.println(area);
    }
}

File: 백준/Bronze/9063. 대지/대지.cc
#include <iostream>
using namespace std;

int main() {

	int n;
	int x, y;
	int minX = 10000;
	int maxX = -10000;
	int minY = 10000;
	int maxY = -10000;

	cin >> n;

	for (int i = 0; i < n; i++)
	{
		cin >> x >> y;
		if (x < minX)
		{
			minX = x;
		}
		if (x > maxX)
		{
			maxX = x;
		}
		if (y < minY)
		{
			minY = y;
		}
		if (y > maxY)
		{
			maxY = y;
		}
	}
	
	int area = (maxX - minX) * (maxY - minY);

	cout << area << endl;
}